plugins/ua_network_tcp.c

```diff
@@ -291,12 +291,6 @@ ServerNetworkLayerTCP_closeConnection(UA_Connection *connection) {
         return;
     connection->state = UA_CONNECTION_CLOSED;
 #endif
-#if UA_LOGLEVEL <= 300
```

Comment:
I deleted it because I need the pointer for the servent as already discussed with you!